packages/cspell-glob/package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "tsc -p .",
     "watch": "tsc -p . -w",
-    "clean": "rimraf dist",
+    "clean": "rimraf dist coverage",
```

Comment:
All of this can be done at the top level:
`rimraf packages/*/{dist,temp,coverage} integration-tests/dist && lerna run clean"

Reply:
Yup, would probably need something for the test folders too, but that might be a better idea